src/main/java/net/mcreator/element/parts/gui/Label.java

```diff
@@ -82,4 +95,7 @@ public String getRenderText() {
 		}
 	}
 
+	@Override public AnchorPoint getAnchorPoint() {
+		return this.anchorPoint;

```

Comment:
(`this.` is not needed in this case as the method does not have a variable that is also called `anchorPoint`)

Reply:
LGTM